data/crd/ssp.kubevirt.io_ssps.yaml

```diff
@@ -14,7 +14,11 @@ spec:
     singular: ssp
   scope: Namespaced
   versions:
-  - name: v1beta1
+  - additionalPrinterColumns:
```

Comment:
Shouldn't that be part of https://github.com/kubevirt/ssp-operator/pull/328?

Reply:
yes, but we forgot about it.